### ISSUE TITLE
Possibility to make sticky windows stay on their tag

### ISSUE THREAD

**Issue by hertg** [leftwm/leftwm#1084]
### Describe the enhancement

Currently when I mark a window as sticky, it effectively moves the window to the tag I'm on every time I'm switching. When I make a window "unsticky" it stays wherever it is at this point in time. I'd prefer it to be sticky as in "stay visible, but don't actually move to different tags". 
I believe that is how sticky windows in `dwm` behave, which is what I used a few years ago before LeftWM, I still haven't gotten used to LeftWM's sticky behavior. I'd even propose to change the default behavior, but I am also happy with a configuration change that allows to choose the preferred way.

### How important is this enhancement to you?

My workflow is impaired without it

### How important do you think is this enhancement for the average LeftWM user?

Would probably be nice to have

**Comment by VuiMuich** [comment]
So you would prefer that on `unsticky` it would jump back to the tag, where it was when it became sticky?

I would be ok with this being default.

**Comment by hertg** [comment]
> So you would prefer that on `unsticky` it would jump back to the tag, where it was when it became sticky?

Exactly. For example when I make my browser sticky, I do so because I'm looking at something on the internet that I want to see constantly. When I no longer need to see it constantly, I make it unsticky. At this point it doesn't matter where I currently am, I want the browser to still be in the `surf` tag where it's supposed to be, not on the tag wherever I decided that it is no longer useful to carry around. Sometimes I also make stuff unsticky because I simply run out of space to carry semi-important stuff around, I do so on tags where the space runs out and I want the window gone. This makes the current default non-optimal in most of the cases where I `unsticky` a window.

**Comment by VuiMuich** [comment]
A pretty simple but only partial solution would be to check on unsticks if window rules apply to the window. But this leaves generic windows like terminals out of the loop.

Implementation wise the full deal might be a bit trickier, as I believe to sticky a window we simply subscribe it to all tags, right?
But iirc we have a `tag_history`, and we could forbid updating it, when the window is stickied, so we find can easily find back to the tag where it was before.

**Comment by hertg** [comment]
I don't think this needs immediate attention, I just started to document some stuff that's been annoying me for a while. This, #1078, and #1079 are something I'd like to take a look at if I were to take on some refactoring in workspace/tag/window logic. I'd like to do that, similar to what I did with the layouts, but I can't make any promises at this time.
